client/src/app/all-projects/all-projects.component.ts

```diff
@@ -43,7 +47,7 @@ export class AllProjectsComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.dataSource = new ProjectDataSource(this.getProjects.bind(this));
     this.dataSource.sortBy('updateDate', 'desc');
-    this.dataSource.connect(true).subscribe({
+    this.dataSource.connect(true, THIRTY_SECONDS).subscribe({
```

Comment:
but what is THIRTY_SECONDS. Is it the value of the timeout?

Reply:
Correct, could be improved.